### PR TITLE
Add download url(s) for kubectl

### PR DESCRIPTION
**Pull request by dims**
Copy some download info from http://kubernetes.io/docs/getting-started-guides/minikube/#install-kubectl

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/1458)

### CODE REVIEW

File: docs/user-guide/prereqs.md [hunk 1]
```diff
@@ -7,11 +7,41 @@ assignees:
...
 
-## Installing kubectl
+## Downloading kubectl
+
+If you built from source, kubectl should be either under `_output/local/bin/<os>/<arch>` or `_output/dockerized/bin/<os>/<arch>`.
 
 If you downloaded a pre-compiled [release](https://github.com/kubernetes/kubernetes/releases), kubectl should be under `platforms/<os>/<arch>` from the tar bundle.
 
-If you built from source, kubectl should be either under `_output/local/bin/<os>/<arch>` or `_output/dockerized/bin/<os>/<arch>`.
+You may also download a pre-built kubectl client binary for say `${K8S_VERSION}` (in this example: `{{page.version}}.0`) for specific platforms:
```
Comment (janetkuo) [inline docs/user-guide/prereqs.md]: @devin-donnelly what should `{{page.version}}` give me? Right now it gives me `1.3`

File: docs/user-guide/prereqs.md [hunk 2]
```diff
@@ -7,11 +7,41 @@ assignees:
...
+
+```shell
```
Comment (janetkuo) [inline docs/user-guide/prereqs.md]: This part is the same as http://kubernetes.io/docs/getting-started-guides/minikube/#install-kubectl

@devin-donnelly should we duplicate it or put this in a centralized place?

File: docs/user-guide/prereqs.md [hunk 3]
```diff
@@ -7,11 +7,41 @@ assignees:
...
+# OS X/386 
+curl -Lo kubectl http://storage.googleapis.com/kubernetes-release/release/{{page.version}}.0/bin/darwin/386/kubectl && chmod +x kubectl && sudo mv kubectl /usr/local/bin/
+```
+
+The generic download path is:
+```
+https://storage.googleapis.com/kubernetes-release/release/${K8S_VERSION}/bin/${GOOS}/${GOARCH}/${K8S_BINARY}
+```
+
+Windows users can use `windows/amd64` or `windows/386` for `${GOOS}/${GOARCH}` in the download url.
```
Comment (janetkuo) [inline docs/user-guide/prereqs.md]: Just translate this in the shell example?

**Comment by janetkuo** [comment]
Finished my review. The rest needs writer's review

**Comment by janetkuo** [comment]
The second commit seems to be empty?

**Comment by janetkuo** [comment]
Ref #1500

File: docs/user-guide/prereqs.md (hunk 1 above)
Comment (devin-donnelly) [inline docs/user-guide/prereqs.md]: I don't know where that variable comes from. I'll figure it out.

File: docs/user-guide/prereqs.md (hunk 3 above)
Comment (dims) [inline docs/user-guide/prereqs.md]: @janetkuo - did not want to assume curl availability for windows users :)

**Comment by devin-donnelly** [comment]
@dims, @janetkuo, I don't see any new commits to address Janet's feedback. I can't merge this until those comments/changes are resolved.

File: docs/user-guide/prereqs.md (hunk 2 above)
Comment (mattfarina) [inline docs/user-guide/prereqs.md]: Are you suggesting this move to a page on how to download `kubectl`? Then minikube, pre-reqs, and others can link to it?

File: docs/user-guide/prereqs.md (hunk 3 above)
Comment (mattfarina) [inline docs/user-guide/prereqs.md]: The windows url ends in `.exe`. It might be worth explicitly pointing that out somehow.

Comment (janetkuo) [inline docs/user-guide/prereqs.md]: oh, right :)

Comment (jaredbhatti) [inline docs/user-guide/prereqs.md]: This was fixed in a separate PR: https://github.com/kubernetes/kubernetes.github.io/pull/1500

**Comment by devin-donnelly** [comment]
@dims , can you fix the merge conflict?

**Comment by jaredbhatti** [comment]
@dims Heya, can you merge the conflicts?

**Comment by chenopis** [comment]
@dims This PR looks stale since it's over 4 months old, so I'm going to close it for now. Please reopen it when you get a chance to resolve the conflicts. Thx

**Pull request closed without merging**